spec/features/helpers/sessions.rb

```diff
@@ -0,0 +1,20 @@
+module SessionHelpers
+
+  def register(user)
+    visit '/user/register'
+    expect(page.status_code).to eq(200)
+    fill_in :email,    with: user.email
+    fill_in :password, with: user.password
+    fill_in :username, with: user.username
+    fill_in :password_confirmation, with: user.password_confirmation
+    click_button 'Sign up for Chitter'
+  end
+
+  def sign_in(email:, password:)
```

Comment:
Unused method argument - `email`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.